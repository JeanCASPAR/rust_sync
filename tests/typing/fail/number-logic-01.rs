use rust_sync::sync;

sync! {
    #![pass(1)]

    node oui() = (a)
    where
        a : bool = !3;
}

fn main() {}
