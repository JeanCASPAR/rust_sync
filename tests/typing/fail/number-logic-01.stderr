error: type mismatch: expected `bool`, found `int`
 --> tests/typing/fail/number-logic-01.rs:6:20
  |
6 |         a : bool = !3;
  |                    ^^
